# Header-less PointCloud2
# Based on sensor_msgs/PointCloud2

uint32 height
uint32 width
sensor_msgs/PointField[] fields
bool is_bigendian
uint32 point_step
uint32 row_step
uint8[] data
bool is_dense